src/utils/filePath.ts

```diff
@@ -0,0 +1,56 @@
+import * as R from 'ramda'
```

Comment:
@quesabe please rename the file to `file-path.ts`. We follow the dash case in file names. Camel case only used in components dirs names.

Reply:
Sure. Sorry, butI always forget about this rule.